examples/dft/compile_time_dispatching/complex_fwd_buffer_mklgpu.cpp

```diff
@@ -55,7 +55,7 @@ void run_example(const sycl::device& gpu_device) {
     // 1. create descriptors
     oneapi::mkl::dft::descriptor<oneapi::mkl::dft::precision::SINGLE,
                                  oneapi::mkl::dft::domain::COMPLEX>
-        desc(N);
+        desc(static_cast<std::int64_t>(N));
```

Comment:
Why can't we just start with N as a `std::int64_t`?

Reply:
`N` is used twice to create `std::vector` which gives a warning if the size isn't unsigned. I figured it's best to avoid the risk of having a negative `N` which would result in a huge allocation on the host (well probably fail without allocating anything)